sdk/identity/src/token_credentials/default_credentials.rs

```diff
@@ -14,7 +14,11 @@ pub struct DefaultAzureCredentialBuilder {
 impl DefaultAzureCredentialBuilder {
     /// Create a new `DefaultAzureCredentialBuilder`
     pub fn new() -> Self {
-        Self::default()
+        Self {
```

Comment:
I would provide an `impl Default for DefaultAzureCredentialBuilder`, since `new()` takes no parameters anyway.

Reply:
Done